runtime/src/test_common.rs

```diff
@@ -0,0 +1,142 @@
+//! Boilerplate for runtime module unit tests
+
+use crate::did;
+use crate::did::{Did, DidSignature};
+use crate::revoke::{Policy, RegistryId, RevokeId};
+use codec::Encode;
+use frame_support::{impl_outer_origin, parameter_types, weights::Weight};
+use sp_core::{Pair, H256};
+use sp_runtime::{
+    testing::Header,
+    traits::{BlakeTwo256, IdentityLookup},
+    Perbill,
+};
+
+pub use frame_support::dispatch::DispatchError;
+pub use rand::random;
+pub use sp_core::sr25519;
+pub use std::iter::once;
+
+pub type RevoMod = crate::revoke::Module<Test>;
+pub type BlobMod = crate::blob::Module<Test>;
```

Comment:
`BlobMod` is not used anywhere else so it should not be added here.

Reply:
Addressed in 99a5659120f7f1238a5f93831f3f9de376932e31.

Moved `BlobMod` declaration into blob test module.